src/routes/clubs.$slug.manage.tsx

```
@@ -661,6 +737,159 @@ export default function ClubManageRoute() {
             )}
             {activeTab === "analytics" && <ClubAnalyticsDashboard clubId={club.id} />}
             {activeTab === "merchandise" && <ManageMerch clubId={club.id} />}
+            {activeTab === "developer" && (
+              <div className="neu-border bg-white p-6 space-y-6">
+                <div className="flex items-center justify-between border-b-2 border-black pb-2">
+                  <h2 className="font-display text-2xl font-bold">
+                    Secure API Key Management
+                  </h2>
```

ESLint

[error] Replace `⏎····················Secure·API·Key·Management⏎··················` with `Secure·API·Key·Management`

(prettier/prettier)